discovery-azure-api/src/main/scala/akka/discovery/azureapi/JsonFormat.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.discovery.azureapi
+
+import akka.annotation.InternalApi
+import akka.discovery.azureapi.PodList._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import spray.json._
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[azureapi] object JsonFormat extends SprayJsonSupport with DefaultJsonProtocol {
```

Comment:
Maybe unit test with some example json to verify this?

Reply:
Yep, I copied it over from `kubernetes-api`. I'll add tests both places if they are missing.